views/pages/practitioner/declaration.njk

```diff
@@ -0,0 +1,354 @@
+{% extends "layouts/layout.njk" %}
+
+{% block content %}
+
+    {{ backButton('dashboard') }}
+
```

Comment:
Can we add a page title? I like: `Declaration and review`

Reply:
I think it would be useful to add the 'Patient:' line in as with other pages too, it's noticeably absent in the print view.

Reply:
done and done